src/nls/root/strings.js

```diff
@@ -63,6 +63,216 @@ define(function (require, exports, module) {
             UNGROUP_LAYERS: "Ungroup layers"
         },
         APP_NAME: "Photoshop",
+        MENU: {
```

Comment:
This is a rebase error. There is no longer a `MENU` property in strings.js. Instead, this has all moved to `i18n!nls/menu`.